Submodels/file_handling.py

```
@@ -0,0 +1,689 @@
+                                            ])
+    
+                            # Write the row to the file
+                            file.write('    '.join(map(str, row)) + '\n')
+                        # End the radial section
+                        file.write('END\n')
+                    # End the stage 
+                    file.write('\nEND\n \n')
+                # End the input file
+                file.write('END\n')
+                        
+            return None
```

Ruff (0.8.2)

Do not explicitly `return None` in function if it is the only possible return value

Remove explicit `return None`

(RET501)